content/en/blog/_posts/2020-01-17-Docs-Review-2019.md

```diff
@@ -0,0 +1,99 @@
+
+SIG Docs continues to solve how best to handle [third-party content](https://github.com/kubernetes/enhancements/pull/1327). **There's too much vendor content on kubernetes.io**, and guidelines for adding or rejecting third-party content remain unclear. The discussion so far has been powerful, including pushback demanding greater collaborative input&mdash;a powerful reminder that Kubernetes is in all ways a communal effort.
+
+
+We're in the middle of our third chair transition in 18 months. Each chair transition has been healthy and collegial, but it's still a lot of turnover in a short time. Chairing any open source project is difficult, but especially so with SIG Docs. Chairship of SIG Docs requires a steep learning curve across multiple domains: docs (both written and generated from spec), information architecture, specialized contribution paths (for example, localization), how to run a release cycle, website development, CI/CD, community management, on and on. It's a role that requires multiple people to function successfully without burning people out. Training replacements is time-intensive.
+
+Perhaps most pressing in the Not So Good category is that SIG Docs currently has only one technical writer dedicated full-time to Kubernetes docs. This has impacts on Kubernetes docs: some obvious, some less so.
+
+## Impacts of understaffing on Kubernetes docs
+
+If Kubernetes continues through 2020 without more technical writers dedicated to the docs, here's what I see as the most likely possibilities.
+
+### But first, a disclaimer
+
+{{< caution >}}
+
+It is very hard to predict, especially the future.
+-Niels Bohr
+
+{{< /caution >}}
+
+
+Some of my predictions are almost certainly wrong, either by scope or degree. While my estimates are often based in community discussions and awareness, any errors are mine alone. 
+
+That said...
+
+### Effects in 2020
+
+Current levels of function aren't self-sustaining. Even with a strong playbook, the release cycle still requires expert support from at least one (and usually two) chairs during every cycle. Without fail, each release breaks processes in new and unexpected ways, and it requires familiarity and expertise to diagnose and resolve the cycle. As chairs continue to cycle&mdash;and to be clear, regular transitions are part of a healthy community&mdash;we accrue the risks associated with a pool that lacks sufficient professional depth and employer support.
+
+Oddly enough, one of the challenges to staffing is that the docs appear good enough. Based on site analytics and survey responses, readers are pleased with the quality of the docs. When folks visit the site, they generally find what they need and behave like satisfied visitors.
+
+The danger is that this will change over time: slowly with occasional losses of function, annoying at first then increasingly critical. The more time passes without adequate staffing, the more difficult and costly fixes will become.
```

Comment:
```suggestion
The danger is that this will change over time: slowly with occasional losses of function, annoying at first, then increasingly critical. The more time passes without adequate staffing, the more difficult and costly fixes will become.
```

Reply:
I'm sufficiently ambivalent about the extra comma to not fight for it. 😅